src/Sulu/Bundle/AdminBundle/Resources/js/components/InfiniteScroller/InfiniteScroller.js

```diff
@@ -96,12 +100,28 @@ export default class InfiniteScroller extends React.PureComponent<Props> {
 
     render() {
         const {
+            total,
+            current,
+            loading,
             children,
+            lastPageReachedText,
         } = this.props;
+        let indicator = null;
+
+        if (loading) {
+            indicator = <Loader />;
+        } else if (current === total && !loading) {
+            indicator = lastPageReachedText;
+        }
 
         return (
             <div ref={this.setRef}>
-                {children}
+                <div>
+                    {children}
+                </div>
```

Comment:
Doesn't both possible `children` already wrap the elements in a div? Then I wouldn't do it here again. In the snapshot there are already 4 nested divs 😕

Reply:
Yes, but if I don't do that it would be possible to mix the loader with children elements.